libandroid-navigation/src/test/java/com/mapbox/services/android/navigation/v5/navigation/NavigationHelperTest.java

```diff
@@ -36,77 +45,57 @@
 import static junit.framework.Assert.assertNotSame;
 import static junit.framework.Assert.assertTrue;
 import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
 
 @RunWith(RobolectricTestRunner.class)
 @Config(constants = BuildConfig.class, sdk = 25)
 public class NavigationHelperTest extends BaseTest {
```

Comment:
Overall same comments pointed out above ☝️

Reply:
`RobolectricTestRunner` is needed to test the milestones being triggered - removing this results in a crash